components/CollapsibleList/index.js

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components';
+import { useState, useRef } from 'react';
+import './icon';
+import Arrow from './icon';
+const ListSection = styled.div`
+.flex;
+.flex-col;
+`;
+const ListButton = styled.button`
+.bg-gray-300;
+.flex;
+.items-center;
+border: none;
+outline: none;
+width:100%;
+.mt-2;
+.px-5;
+&:hover {
+  background-color: #cbd5e0;
+  cursor:pointer;
+}
+`;
+const ButtonTitle = styled.p`
+.text-gray-700;
+.text-lg;
+@media(max-width:500px){
```

Comment:
same way you can use `.lg:text-lg; text-sm;` This will do exactly the same. For large and xl screens its large text and for small screens small font-size